kivy/uix/screenmanager.py

```diff
@@ -964,6 +964,7 @@ def _get_screen_names(self):
     '''
 
     def __init__(self, **kwargs):
+        self.transition = SlideTransition()
```

Comment:
I would suggest using: 
```
if 'transition' not in kwargs:
    kwargs['transition'] = SlideTransition()
```
This way if `transition` keyword is used in creating of `ScreenManager` instance then default `SlideTransition` instance doesn't get created.

Reply:
Agreed my good friend, thanks for the idea! :-)